DasharoPayloadPkg/Library/FmpDeviceSmmLib/FmpDeviceSmmLib.c

```diff
@@ -1009,6 +1009,17 @@ FmpDeviceSetImageWithStatus (
       continue;
     }
 

```

Comment:
To not leave this information only on GitHub:

```suggestion
    // This assumes that the protected ranges are aligned to SMMSTORE's block size,
    // otherwise the start/end of unprotected ranges might not get updated.
```

Note that this is not granularity of a flash chip, the value is [hard-coded](https://github.com/Dasharo/coreboot/blob/43c572154ea74bbe24b6d67f0ebbb9498499e0e7/src/include/smmstore.h#L41-L47) to be 64 KiB in coreboot.  Is this a problem for `SI_GBE` then?
```
	SI_ALL 10M {
		SI_DESC 16K
#if CONFIG_MAINBOARD_USES_IFD_GBE_REGION
		SI_GBE 8K
#endif
		SI_ME
	}

```

Reply:
Hmm thankfully we don't need to touch this region. Currently we're migrating it but not touching it should work too. I'll add it to the list in IsRangeWriteable

Reply:
https://github.com/Dasharo/edk2/commit/31b804f996ad851a20838d3d7d4179840466ea01

Reply:
LGTM, SI_GBE is in the exclusion list in IsRangeWriteable and we're skipping any overlapping blocks when DescriptorLocked is true.